package.json

```diff
@@ -15,6 +15,7 @@
     "crc": "3.4.4",
     "debug": "2.6.8",
     "depd": "~1.1.0",
+    "memorystore": "~1.2.4",
```

Comment:
Remove the `~` from the version here; the third-party deps are always exact versions.